doc/news/DM-46179.feature.rst

```diff
@@ -0,0 +1,2 @@
+Extend TCS readiness check to other image types beyond OBJECT.
```

Comment:
Please, make the news fragment more relevant explaining the changes you are introducing.